### PR TITLE
MAINT(docker): Make Dockerfile working again

### LINKED ISSUE #4813: Build docker image of 1.3.x
**Describe the bug**
Docker file of branch 1.3.x reference ubuntu:disco wich is not supported anymore.

**Steps to Reproduce**
1. Checkout branch 1.3.x
2. `docker build -t mumble13x .`

```
Step 3/16 : RUN apt-get update && apt-get install -y 	build-essential 	pkg-config 	qt5-default 	libboost-dev 	libasound2-dev 	libssl-dev 	libspeechd-dev 	libzeroc-ice-dev 	libpulse-dev 	libcap-dev 	libprotobuf-dev 	protobuf-compiler 	protobuf-compiler-grpc 	libprotoc-dev 	libogg-dev 	libavahi-compat-libdnssd-dev 	libsndfile1-dev 	libgrpc++-dev 	libxi-dev 	libbz2-dev 	qtcreator
 ---> Running in 9f26d3b69dc1
Ign:1 http://security.ubuntu.com/ubuntu disco-security InRelease
Err:2 http://security.ubuntu.com/ubuntu disco-security Release
  404  Not Found [IP: 91.189.91.39 80]
Ign:3 http://archive.ubuntu.com/ubuntu disco InRelease
Ign:4 http://archive.ubuntu.com/ubuntu disco-updates InRelease
Ign:5 http://archive.ubuntu.com/ubuntu disco-backports InRelease
Err:6 http://archive.ubuntu.com/ubuntu disco Release
  404  Not Found [IP: 91.189.88.152 80]
Err:7 http://archive.ubuntu.com/ubuntu disco-updates Release
  404  Not Found [IP: 91.189.88.152 80]
Err:8 http://archive.ubuntu.com/ubuntu disco-backports Release
  404  Not Found [IP: 91.189.88.152 80]
Reading package lists...
E: The repository 'http://security.ubuntu.com/ubuntu disco-security Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu disco Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu disco-updates Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu disco-backports Release' does not have a Release file.
```

**Expected behavior**
A successful build

**Fix**
Use `ubuntu:latest` as base image like in `master` branch. I'll make a pull-request as soon as I finish testing.

### PR DESCRIPTION
**Pull request by St-Ex-savadenn**
In doing so the current state of the Dockerfile on the master
branch has been used as a template.

Fixes #4813 

### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

**Comment by Krzmbrzl** [comment]
Your commit does quite a few things more than only updating the base image.
Could you explain why the other changes are necessary? :thinking: 

Note also that you commit does _not_ follow our [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md). 

@davidebeatrici I assume "backporting" this to the 1.3.x branch should be fine, right?

**Comment by Krzmbrzl** [comment]
> I used base image and libraries from master Dockerfile

Ah so you basically copied over the Dockerfile from master (except the part that actually builds the code of course) and backported that to 1.3.x?

**Comment by St-Ex-savadenn** [comment]
Hi, I mainly use [master dockerfile](https://github.com/mumble-voip/mumble/blob/master/Dockerfile) as an exemple. I will redo my commit message to meet the expected one.

The image is built on this Gitlab project. I intend to do a daily build and pushed it onto dokcer hub. I ran some tests on this image 
* registry.gitlab.com/savadenn-public/mumble/1.3:latest
* Project: https://gitlab.com/savadenn-public/mumble

**Comment by Krzmbrzl** [comment]
In that case I suggest a commit message like this:
```
MAINT(docker): Make Dockerfile working again

In doing so the current state of the Dockerfile on the master
branch has been used as a template.

Fixes #4813 
```

**Comment by St-Ex-savadenn** [comment]
Done, thanks for the wording.

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,8 @@ RUN apt-get update && apt-get install -y \
 	libgrpc++-dev \
 	libxi-dev \
 	libbz2-dev \
-	qtcreator
+	qtcreator \
```
Comment (Krzmbrzl) [inline Dockerfile:27]: Unless you have found this to be essential, you can remove this line as was done on master as well.

Comment (St-Ex-savadenn) [reply Dockerfile:27]: * [ ] Done

**Review by Krzmbrzl — APPROVED** [review]

**Review by Krzmbrzl — CHANGES_REQUESTED** [review]
Ah hold on a sec. Please squash your two commits into a single one :point_up:

**Comment by St-Ex-savadenn** [comment]
Done

**Review by Krzmbrzl — APPROVED** [review]

**Review by davidebeatrici — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
Thanks for fixing this 👍

**Pull request merged**